package.json

```diff
@@ -4,7 +4,7 @@
   "description": "app_name description",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "echo",
```

Comment:
Revert this change. Just modify this file:

`.husky/pre-commit`

to this: 
```
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npm run lint
```

Reply:
Done!!!!